### PR TITLE
dillo: add page

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/dillo.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> A lightweight browser intended for slow computers.
+> More information: <https://www.dillo.org/>.
+
+- Open Dillo:
+
+`dillo`
+
+- Open Dillo at a specific size and screen location (format: width, height, x_placement and y_placement):
+
+`dillo --geometry {{500x500+250+125}}`
```
Comment (bl-ue) [inline pages/common/dillo.md:12]:
```suggestion
- Open Dillo with a specific window size and screen location:

`dillo --geometry {{width}}x{{height}}+{{x_position}}+{{y_position}}`
```

File: pages/common/dillo.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Open Dillo and show a file or directory:
+
+`dillo {{path/to/file_or_directory}}`
+
+- Open Dillo at full-screen:
+
+`dillo --fullwindow`
```
Comment (bl-ue) [inline pages/common/dillo.md:20]:
```suggestion
- Open Dillo in full-screen mode:

`dillo --fullwindow`
```

**Review by bl-ue — COMMENTED** [review]
A few suggestions.

**Review by bl-ue — APPROVED** [review]

File: pages/common/dillo.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Open Dillo:
+
+`dillo`
+
+- Open Dillo with a specific window size and screen location:
+
+`dillo --geometry {{width}}x{{height}}+{{x_position}}+{{y_position}}`
+
+- Open Dillo and show a file or directory:
```
Comment (marchersimon) [inline pages/common/dillo.md:14]: Wait, is this a web or file browser? I think if you just write `browser` most will assume it's a web browser.

Comment (CleanMachine1) [reply pages/common/dillo.md:14]: Its a browser however just like browsers such as firefox and chrome, it open files

Comment (marchersimon) [reply pages/common/dillo.md:14]: So when you open a directory it opens just all files in that directory?

Comment (marchersimon) [reply pages/common/dillo.md:14]: Also I'd still say `web browser` in the description to avoid confusion

Comment (CleanMachine1) [reply pages/common/dillo.md:14]: Read the description.
I am also going to add another command for URLs

File: pages/common/dillo.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Open Dillo in full-screen mode:
+
+`dillo --fullwindow`
+
+- Display Dillo version:
+
+`dillo --version`
+
+- Display Dillo help:
```
Comment (navarroaxel) [inline pages/common/dillo.md:30]: I think we don't need `Dillo` in every example
```suggestion
- Display help:
```

File: pages/common/dillo.md (hunk 3 above)
Comment (marchersimon) [reply pages/common/dillo.md:18]: > Read the description.

What do you mean? The problem is that the description alone doesn't tell me if it's a file or web browser.

Comment (CleanMachine1) [reply pages/common/dillo.md:18]: > A lightweight browser intended for slow computers.

Comment (CleanMachine1) [reply pages/common/dillo.md:18]: Should we add a file or browser before that?

Comment (marchersimon) [reply pages/common/dillo.md:18]: @bl-ue / @navarroaxel any opinions?

Comment (navarroaxel) [reply pages/common/dillo.md:18]: Like `firefox` and `chromium`, I think the command description should say `web browser`.

Comment (CleanMachine1) [reply pages/common/dillo.md:18]: Ok

**Review by einverne — APPROVED** [review]
Thank you @CleanMachine1 for the new page. Looks good to me.

**Comment by CleanMachine1** [comment]
Was just testing my rebasing skills and it worked!

**Comment by CleanMachine1** [comment]
Merge?

File: pages/common/dillo.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Open Dillo and show a file or directory:
+
+`dillo {{path/to/file_or_directory}}`
+
+- Open Dillo in full-screen mode:
+
+`dillo --fullwindow`
+
+- Display Dillo version:
```
Comment (navarroaxel) [inline pages/common/dillo.md:26]: I think we should remove `Dillo` from the example descriptions.
```suggestion
- Display version:
```

File: pages/common/dillo.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Open Dillo:
+
+`dillo`
+
+- Open Dillo with a specific window size and screen location:
+
+`dillo --geometry {{width}}x{{height}}+{{x_position}}+{{y_position}}`
+
+- Open Dillo at a specific URL:
```
Comment (navarroaxel) [inline pages/common/dillo.md:14]:
```suggestion
- Launch Dillo and open a specific URL:
```

File: pages/common/dillo.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# dillo
+
+> A lightweight web browser intended for slow computers.
+> More information: <https://www.dillo.org/>.
+
+- Open Dillo:
+
+`dillo`
+
+- Open Dillo with a specific window size and screen location:
```
Comment (navarroaxel) [inline pages/common/dillo.md:10]:
```suggestion
- Launch Dillo with a specific window size and screen location:
```

File: pages/common/dillo.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
+# dillo
+
+> A lightweight web browser intended for slow computers.
+> More information: <https://www.dillo.org/>.
+
+- Open Dillo:
```
Comment (navarroaxel) [inline pages/common/dillo.md:6]:
```suggestion
- Launch Dillo:
```

File: pages/common/dillo.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+
+- Open Dillo with a specific window size and screen location:
+
+`dillo --geometry {{width}}x{{height}}+{{x_position}}+{{y_position}}`
+
+- Open Dillo at a specific URL:
+
+`dillo {{duckduckgo.com}}`
+
+- Open Dillo and show a file or directory:
```
Comment (navarroaxel) [inline pages/common/dillo.md:18]:
```suggestion
- Launch Dillo and open a file or directory:
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/dillo.md [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+
+- Launch Dillo and open a specific URL:
+
+`dillo {{duckduckgo.com}}`
+
+- Launch Dillo and open a file or directory:
+
+`dillo {{path/to/file_or_directory}}`
+
+- Open Dillo in full-screen mode:
```
Comment (CleanMachine1) [inline pages/common/dillo.md:22]:
```suggestion
- Launch Dillo in full-screen mode:
```

Comment (CleanMachine1) [reply pages/common/dillo.md:22]: @navarroaxel

**Review by marchersimon — APPROVED** [review]

**Pull request merged**